solvers/blitz.py

```diff
@@ -7,25 +7,22 @@
 
 class Solver(BaseSolver):
     name = 'Blitz'
-    stop_strategy = 'tolerance'
+    stop_strategy = 'iteration'
 
     install_cmd = 'conda'
     requirements = [
-        'pip:git+https://github.com/tommoral/blitzl1.git@FIX_setup_deps'
+        'pip:git+https://github.com/tbjohns/blitzl1.git@master'
     ]
 
     def set_objective(self, X, y, lmbd):
         self.X, self.y, self.lmbd = X, y, lmbd
 
-        # n_samples = self.X.shape[0]
-        # self.lmbd /= n_samples
-
         blitzl1.set_use_intercept(False)
         self.problem = blitzl1.LogRegProblem(self.X, self.y)
+        blitzl1.set_tolerance(0)
```

Comment:
Nitpick: set all global parameters at the same place. I would do this before instanciating the object but if it needs to be after, I would also move the `use_intercept` one.

```suggestion
        blitzl1.set_tolerance(0)
        self.problem = blitzl1.LogRegProblem(self.X, self.y)
```

Reply:
I agree it's better like you suggest !